specifyweb/frontend/js_src/lib/components/Formatters/formatters.ts

```diff
@@ -158,11 +161,18 @@ async function formatField(
 
   let formatted: string | undefined = undefined;
   const hasPermission = hasPathPermission(fields ?? [], 'read');
-  if (hasPermission) {
-    const data = await fetchDistantRelated(parentResource, fields);
+  const data = await fetchDistantRelated(parentResource, fields);
```

Comment:
wouldn't this throw an exception if `!hasPermission`?

Reply:
responding to the messages you sent me in slack:
> Isn't it like it was in 9.3?
before, we had a permission check:
`const hasPermission = hasPathPermission(fields ?? [], 'read');`
and then only if `hasPermission`, was fetchDistantRelated called
where as with your modification, fetchDistantRelated is always called

Reply:
> It doesn't throw an error.

~~Could you please double check the network or the console tab?
We have the logic for when one error occurs, other related errors (like permission errors) are suppressed so as not to overwhelm the user needlessly.
Or, can you tell me how to replicate the permission error you are encountering?~~

I double-checked the code, and this code change does indeed introduce a bug.
In the case of this dialog, the current collection and current user resource are provided by the back-end even if user does not have read permission to specify user and collection table (because this bare minimum is necessary for things to work).
And in this dialog, the formatter is invoked to just get collectionName field from Collection and user name from SpecifyUser - since these resources are already fetched, and no requests to related tables are happening, no extra network requests are happening, and so no permission error is occurring.

But if you try to format a resource, where the formatter accesses resources the user has no permission to (i.e try to format Determinaion, when user has no read access to Taxon), you would see an error